lib/web_ui/lib/src/engine/clipboard.dart

```diff
@@ -109,7 +109,7 @@ class ClipboardAPICopyStrategy implements CopyToClipboardStrategy {
   @override
   Future<bool> setData(String? text) async {
     try {
-      await html.window.navigator.clipboard.writeText(text!);
+      await html.window.navigator.clipboard!.writeText(text!);
```

Comment:
Sounds like we no longer need to use `_unsafeIsNull` on line 95 to check for clipboard presence. Also, I imagine `window.navigator.clipboard` is a singleton, and so we can store it in a final non-null field, null check it in the initializer once, and remove the null assertions here and below.

Reply:
I think we still want to check whether it's null in order to determine what "Strategy" class to initialize, right?

I don't think `clipboard` can be made a singleton since `_Clipboard` is a private class (unless we declared the field as dynamic).